ozkour-back/domain/model/talk.js

```diff
@@ -1,11 +1,12 @@
 class Talk {
-  constructor (date, universe, eventType, eventName, talkTitle, speakers, checked = true) {
+  constructor (date, universe, eventType, eventName, talkTitle, speakers, link, checked = true) {
```

Comment:
nitpick: il y a trop d'arguments, on en rajoute un "au milieu", il serait pertinent d'utiliser la destructuration pour recuperer les args par nom:
```suggestion
  constructor ({ date, universe, eventType, eventName, talkTitle, speakers, link, checked = true }) {
```